index.md

```diff
@@ -59,6 +59,11 @@ JanusGraph is a project under [The Linux
 Foundation](https://www.linux.com/blog/Linux-Foundation-welcomes-JanusGraph),
 and includes participants from Expero, Google, GRAKN.AI, Hortonworks, and IBM.
 
+## <a name="presentations"></a>Presentations
```

Comment:
The `<a name="...">` is unnecessary because `## Presentations` will automatically create that anchor.